### PR TITLE
Fix not being able to install extensions sometimes

### LINKED ISSUE #4766: Extension Installation Issue
Lens gets stuck on the extension installation 
While trying to install the extension installation lens stucks after pasting the .tgz file or @mirantis/lens-extension-cc 

I had to go to Lens status bar icon and quit app and then restart the app and the extension was there. This is missing from the installation guide.

Environment
- Lens Version: 5.3.4
- OS: Windows 64 
- Installation method (e.g. snap or AppImage in Linux):  npm pack @mirantis/lens-extension-cc

### PR DESCRIPTION
**Pull request by Nokel81**
- Specifically, when there are remnants of a previous install of the
  extension which caused the code to be confused and enter the wrong
  branch

Signed-off-by: Sebastian Malton <sebastian@malton.name>

### CODE REVIEW

**Comment by aleksfront** [comment]
Please try to test with https://github.com/nevalla/lens-resource-map-extension/releases. Inside this branch it's stuck in unpacking step.
freeze in unpacking


(I'm using alpha link on screenshot, but same goes for stable version).

On `master`, unpacking goes okay, however extension won't get installed because of other errors.

**Comment by aleksfront** [comment]
Also, this happens in master also, but can't install extension by its npmjs name
no install from npm

File: src/renderer/components/+extensions/attempt-install/attempt-install.tsx [hunk 1]
```diff
@@ -80,17 +80,12 @@ export const attemptInstall =
       }
 
       const extensionFolder = getExtensionDestFolder(name);
-      const folderExists = await fse.pathExists(extensionFolder);
+      const installedExtensionsion = extensionLoader.getExtension(validatedRequest.id);
```
Comment (jansav) [inline src/renderer/components/+extensions/attempt-install/attempt-install.tsx]: typo 

```suggestion
      const installedExtension = extensionLoader.getExtension(validatedRequest.id);
```

File: src/renderer/components/+extensions/attempt-install/attempt-install.tsx [hunk 2]
```diff
@@ -80,17 +80,12 @@ export const attemptInstall =
...
 
-      if (!folderExists) {
-      // install extension if not yet exists
-        await unpackExtension(validatedRequest, dispose);
-      } else {
-        const {
-          manifest: { version: oldVersion },
-        } = extensionLoader.getExtension(validatedRequest.id);
+      if (installedExtensionsion) {
```
Comment (jansav) [inline src/renderer/components/+extensions/attempt-install/attempt-install.tsx]:
```suggestion
      if (installedExtension) {
```

File: src/renderer/components/+extensions/attempt-install/attempt-install.tsx [hunk 3]
```diff
@@ -80,17 +80,12 @@ export const attemptInstall =
...
+        const { version: oldVersion } = installedExtensionsion.manifest;
```
Comment (jansav) [inline src/renderer/components/+extensions/attempt-install/attempt-install.tsx]:
```suggestion
        const { version: oldVersion } = installedExtension.manifest;
```

**Review by jansav — CHANGES_REQUESTED** [review]

**Comment by Nokel81** [comment]
> 

@aleksfront Using the latest `.tgz` file link I was able to install easily and without too much delay.

**Comment by Nokel81** [comment]
@jansav PTALA, I have fixed the typo

**Comment by aleksfront** [comment]
@Nokel81 I don't know the exact reason but it's still stuck in `unpacking` step for me. However, it seems that some files appeared in `.k8slens/extensions` folder. See below:

https://user-images.githubusercontent.com/9607060/152473691-4bbe93cf-cb98-4078-8c52-ead57c568441.mov

**Review by jansav — APPROVED** [review]

**Review by ixrock — APPROVED** [review]

**Pull request merged**